ravenframework/Optimizers/GeneticAlgorithm.py

```diff
@@ -587,40 +682,54 @@ def _useRealization(self, info, rlz):
                                      self.rank,
                                      self.crowdingDistance,
                                      self.objectiveVal,
-                                     self.constraints,
+                                     self.fitness,
                                      self.constraintsV)
           self._resolveNewGenerationMulti(traj, rlz, info)
```

Comment:
self. constraintsV is not so descriptive, is it constrainedVariables? if so maybe 'self.constrainedVars'?

Reply:
descriptions of self.xxxx are now added in __init__ method.